nodejs/.env

```diff
@@ -0,0 +1,19 @@
+DB_USER=lora
+DB_PASSWORD=P@ssw0rd
```

Comment:
password can be seen!!!!!!!!!!! we need to hash or have it in git hub secrets

Reply:
Can we get rid of the whole nodejs folder?

Reply:
It has been deleted